### ISSUE TITLE
Locked out of admin instance - phoenix.js:844 WebSocket connection failed

### ISSUE THREAD

**Issue by heaversm** [mozilla/hubs-cloud#202]
I set up a hubs cloud instance and was trying to add CORS domains (unsuccessfully) in order to insert a custom script into my home page `<head>`. I was still getting CORS errors, and made some sort of change to that field and my admin console would no longer render. I'm now getting this error in the browser:

`phoenix.js:844 WebSocket connection to 'wss://[mydomain.com]/socket/websocket?vsn=2.0.0' failed:`

I can access the signin page, and when I enter my email to get a login link, I never receive anything.

<img width="1057" alt="Screen Shot 2021-06-02 at 5 00 42 PM" src="https://user-images.githubusercontent.com/520541/120566647-89edc700-c3c4-11eb-9c00-8c174d1e6507.png">

**Comment by gdmtech** [comment]
Same issue for us
template 1.1.2

**Comment by alfainfotech** [comment]
Same issue for us too 
template 1.1.2

**Comment by alfainfotech** [comment]
> @alfainfotech same issue for me. any solution?

Yes, I resolved my issue by deleting that stack. And reinstalling hubs cloud from fresh resolved the issue. But still I'm curious to know what if that error returns. I guess solution or feedback needs to come from hubs team support.

**Comment by nupur711** [comment]
Thank you so much for the quick reply...How did you delete that stack? Reinstalling hubs cloud means the whole server or pulling the repository to the VS code?

**Comment by alfainfotech** [comment]
I'm not sure where you are working, I installed mine on AWS so I just deleted whole stack application and redo from beginning.

**Comment by nupur711** [comment]
Okay. I am using AWS only. Does it take too much of time to reinstallation? I am asking this because there is different team who is taking care of AWS server.
